src/OpenToolkit.Mathematics/Color/Color3.cs

```diff
@@ -0,0 +1,1231 @@
+        /// </summary>
+        public static Color3<Rgb> Springgreen => new Color3<Rgb>(0f, 1f, 0.49803922f);
+
+        /// <summary>
+        /// Gets system color steelblue corresponding to 70 130 180 (0.27450982, 0.50980395, 0.7058824).
+        /// </summary>
+        public static Color3<Rgb> Steelblue => new Color3<Rgb>(0.27450982f, 0.50980395f, 0.7058824f);
+
+        /// <summary>
+        /// Gets system color tan corresponding to 210 180 140 (0.8235294, 0.7058824, 0.54901963).
+        /// </summary>
+        public static Color3<Rgb> Tan => new Color3<Rgb>(0.8235294f, 0.7058824f, 0.54901963f);
+
+        /// <summary>
+        /// Gets system color teal corresponding to 0 128 128 (0, 0.5019608, 0.5019608).
+        /// </summary>
+        public static Color3<Rgb> Teal => new Color3<Rgb>(0f, 0.5019608f, 0.5019608f);
+
+        /// <summary>
+        /// Gets system color thistle corresponding to 216 191 216 (0.84705883, 0.7490196, 0.84705883).
+        /// </summary>
+        public static Color3<Rgb> Thistle => new Color3<Rgb>(0.84705883f, 0.7490196f, 0.84705883f);
+
+        /// <summary>
+        /// Gets system color tomato corresponding to 255 99 71 (1, 0.3882353, 0.2784314).
+        /// </summary>
+        public static Color3<Rgb> Tomato => new Color3<Rgb>(1f, 0.3882353f, 0.2784314f);
+
+        /// <summary>
+        /// Gets system color turquoise corresponding to 64 224 208 (0.2509804, 0.8784314, 0.8156863).
+        /// </summary>
+        public static Color3<Rgb> Turquoise => new Color3<Rgb>(0.2509804f, 0.8784314f, 0.8156863f);
+
+        /// <summary>
+        /// Gets system color violet corresponding to 238 130 238 (0.93333334, 0.50980395, 0.93333334).
+        /// </summary>
+        public static Color3<Rgb> Violet => new Color3<Rgb>(0.93333334f, 0.50980395f, 0.93333334f);
+
+        /// <summary>
+        /// Gets system color wheat corresponding to 245 222 179 (0.9607843, 0.87058824, 0.7019608).
+        /// </summary>
+        public static Color3<Rgb> Wheat => new Color3<Rgb>(0.9607843f, 0.87058824f, 0.7019608f);
+
+        /// <summary>
+        /// Gets system color white corresponding to 255 255 255 (1, 1, 1).
+        /// </summary>
+        public static Color3<Rgb> White => new Color3<Rgb>(1f, 1f, 1f);
+
+        /// <summary>
+        /// Gets system color whitesmoke corresponding to 245 245 245 (0.9607843, 0.9607843, 0.9607843).
+        /// </summary>
+        public static Color3<Rgb> Whitesmoke => new Color3<Rgb>(0.9607843f, 0.9607843f, 0.9607843f);
+
+        /// <summary>
+        /// Gets system color yellow corresponding to 255 255 0 (1, 1, 0).
+        /// </summary>
+        public static Color3<Rgb> Yellow => new Color3<Rgb>(1f, 1f, 0f);
+
+        /// <summary>
+        /// Gets system color yellowgreen corresponding to 154 205 50 (0.6039216, 0.8039216, 0.19607843).
+        /// </summary>
+        public static Color3<Rgb> Yellowgreen => new Color3<Rgb>(0.6039216f, 0.8039216f, 0.19607843f);
```

Comment:
Please make them `readonly`

Reply:
These are properties :)

Reply:
Sh.... indeed. Then, can you turn them into `public readonly` fields please?